client/packages/system/src/Stock/Components/StockLineForm.tsx

```diff
@@ -74,6 +75,12 @@ export const StockLineForm: FC<StockLineFormProps> = ({ draft, onUpdate }) => {
     }
   };
 
+  const debouncedUpdate = useDebounceCallback(
```

Comment:
added some debounce, as the form is jumpy for me - the mutating of `draft` is causing a render which is noticeable when a plugin is rendered too